File: benchmark/results/java-invocation-script/run-003/b-baseline/output/com/metaengine/demo/ordering/services/OrderRepository.java
package com.metaengine.demo.ordering.services;

import java.util.List;

import com.metaengine.demo.ordering.aggregates.Order;

/** OrderRepository for the ordering domain. */
public class OrderRepository {

    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-005/b-baseline/output/metaengine_demo/ordering/services/order_repository.py
"""OrderRepository for the ordering domain."""
from typing import Optional

from metaengine_demo.ordering.aggregates.order import Order


class OrderRepository:
    """OrderRepository."""

    def create(self, input: Optional[Order]) -> Order:
        raise NotImplementedError("not implemented")

    def find_by_id(self, id: str) -> Optional[Order]:
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Order]:
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        raise NotImplementedError("not implemented")
